extension/src/popup/components/EnterPassword/index.tsx

```diff
@@ -39,7 +39,7 @@ export const EnterPassword = ({
   const titleLabel = title || t("Enter your password");
   const descriptionLabel = `${
     description || t("Enter your account password to verify your account.")
-  } ${t("You won’t be asked to do this again for the next 24 hours.")}`;
+  }`;
```

Comment:
@CassioMG Heads up that after some more discussion, we decided to remove this line during QA

Reply:
fine by me!